src/features/tokens/routes/types.ts

```diff
@@ -3,22 +3,52 @@ export enum RouteType {
   CollateralToSynthetic = 'collateralToSynthetic',
   SyntheticToSynthetic = 'syntheticToSynthetic',
   SyntheticToCollateral = 'syntheticToCollateral',
+  IbcNativeToIbcNative = 'ibcNativeToIbcNative',
+  IbcNativeToHypNative = 'ibcNativeToHypNative',
```

Comment:
Still don't quite get what `HypNative` is in this context? Don't we usually call `HypNative` the warp contract that takes the native token on the EVM side?

Reply:
I'll call it HypSynthetic.